packages/cfd/build/webpack.config.js

```diff
@@ -38,6 +34,72 @@ module.exports = function (env) {
+            sideEffects: true,
+            usedExports: true,
+            splitChunks: {
+                automaticNameDelimiter: '~',
+                cacheGroups: {
+                    default: {
+                        minChunks: 2,
+                        priority: -20,
+                        reuseExistingChunk: true,
+                    },
+                    defaultVendors: {
+                        chunks: 'all',
+                        idHint: 'vendors',
+                        priority: -10,
+                        test: /[\\/]node_modules[\\/]/,
+                    },
+                    assets: {
+                        chunks: 'all',
+                        name: 'assets',
+                        test: /[\\/]Assets[\\/]/,
+                        enforce: true,
+                    },
+                    internalCFDStoreUsage: {
+                        name: 'internalCFDStoreUsage',
+                        test: module => {
+                            const isStoreModule = module.resource && module.resource.includes('Stores');
+                            return isStoreModule;
+                        },
+                        chunks: chunk => chunk.name !== 'CFDStore', // Do not include if chunk is CFDStore
+                        enforce: true,
+                    },
+                    utilities: {
+                        name: 'utilities',
+                        test: module => {
+                            // Access the resource path of the module
+                            const { resource } = module;
+                            if (!resource) return false;
+
+                            // Define inclusion patterns for Helpers and Constants directories
+                            const includePatterns = [
+                                /[\\/]Helpers[\\/]/, // Include Helpers directory
+                                /[\\/]Constants[\\/]/, // Include Constants directory
+                            ];
+
+                            // Define exclusion patterns to ensure other directories are not included
+                            const excludePatterns = [
+                                /[\\/]Stores[\\/]/, // Since there is a Helpers directory inside Stores, the Stores directory has been excluded.
+                            ];
+
+                            // Check if the module should be included
+                            const shouldBeIncluded = includePatterns.some(pattern => pattern.test(resource));
+
+                            // Check if the module should be excluded
+                            const shouldBeExcluded = excludePatterns.some(pattern => pattern.test(resource));
+
+                            return shouldBeIncluded && !shouldBeExcluded;
+                        },
+                        chunks: 'all',
+                        enforce: true,
+                    },
+                },
+                minChunks: 1,
+                minSize: 35000,
```

Comment:
just curious, how did we get this size?

Reply:
A few modules, such as assets, are slightly over 35kb, which I wanted to combine into a chunk.